### ISSUE TITLE
Picking AAC Encoder

### ISSUE THREAD

**Issue by xtemp09** [lordmulder/LameXP#13]
If there are more than one AAC encoders available, the program detects them all, but offers only one. I think, it would be more suitable if the program offered a checkbox group to choose what AAC encoder to use (on Compression tab, perhaps).

LameXP offers them in the following order: qaac.exe > fhgaacenc.exe > neroAacEnc.exe. The console debug output writes approximately: "Found Nero AAC <…> Enabled Nero AAC <…> Found QAAC <…> Enabled qaac," but offers QAAC without notifying user.

P.S. Qaac64 is ignored. Maybe qaac64 should be used if the system is 64-bit?

**Comment by lordmulder** [comment]
I may consider adding a Combobox for choosing the AAC encoder. But this would require quite a few changes under the hood, so don't expect it soon. Anyway, since AAC encoders need to be installed separately, why would you install an AAC encoder that you don't want to use? Only use case I see for switching between different AAC encoders frequently would be when you want to evaluate various AAC encoders (e.g. for listening tests). But how many users do that? And would the "audiophiles" who do this kind of evaluation use tool like LameXP for that purpose?

**Comment by xtemp09** [comment]
Agreed, but I would express it in a different way - to switch encoders is helpful since they have different quality and different parameters such as 2-pass encoding or different resampling algorithms. The latter is still throwing me into confusion, since sample rate reducing is unpredictable or requires some parameters to be set. I think I can get to the bottom of this with additional command-line options in custom encoder parameters.
